src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -247,7 +250,7 @@ export class SbbDatepicker implements ComponentInterface {
           this._inputElement?.max
         )
       ) {
-        this._inputElement.classList.add('sbb-invalid');
+        this._inputElement.dataset.sbbInvalid = 'true';
```

Comment:
better use toggleDatasetEntry helper method or else assign = ''. true is not needed.